src/gateway/get-started/configure-services-and-routes.md

```diff
@@ -1,5 +1,215 @@
+  --url localhost:8001/services/example_id \
+  --data retries=6
+```
+
+The response body will display the updated change:
+
+```sh
+{
+	"updated_at": 1660065439,
+	"path": null,
+	"connect_timeout": 60000,
+	"write_timeout": 60000,
+	"id": "example_id",
+	"tls_verify": null,
+	"host": "mockbin.org",
+	"tls_verify_depth": null,
+	"created_at": 1660063550,
+	"name": "example_service1",
+	"retries": 6,
+	"client_certificate": null,
+	"read_timeout": 60000,
+	"ca_certificates": null,
+	"port": 80,
+	"protocol": "http",
+	"enabled": true,
+	"tags": null
+}
+```
+
+### Add a route
+
+With an active service you can configure a route to point to that service. The router is what helps {{site.base_gateway}} forward requests to upstream services. Now you will configure a route `/mock` for the service `example_service`. This route will have a specific path that users will use to send requests.
+A route is configured by sending a **POST** request to the [route object](/gateway/latest/admin-api/#route-object) like this:
+
+```sh
+curl -i -X POST http://localhost:8001/services/example_service/routes \
+  --data 'paths[]=/mock' \
+  --data name=mocking
+```
+
+If creating the route was successful the API will return a `201` response code and a response body like this:
+
+```sh
+{
+	"snis": null,
+	"updated_at": 1660067503,
+	"preserve_host": false,
+	"id": "example_service_id",
+	"protocols": [
+		"http",
+		"https"
+	],
+	"created_at": 1660067503,
+	"service": {
+		"id": "example_service"
+	},
+	"hosts": null,
+	"headers": null,
+	"name": "mocking",
+	"tags": null,
+	"paths": [
+		"/mock"
+	]
+}
+```
+
+### Update a route
+
+Routes can be tagged. Tags are an optional set of strings associated with the route that can be used for grouping and filtering. You can assign tags by sending a **PATCH** request to the [services endpoint](/gateway/latest/admin-api/#update-route) and specifying a route. 
+
+```
+curl --request PATCH \
+  --url localhost:8001/services/example_service/routes/mocking \
+  --data tags="tutorial"
+```
+
+In this example, use the `id` field from the service output, and the name of the route to build the request. If the tag was applied successfully the response body will contain the tag. 
+
+```sh
+"tags": [
+		"\"tutorial\""
+	],
+```
+
+You can also use the [routes endpoint](/gateway/latest/admin-api/route) to query for all of the configured routes:
+
+```sh
+curl -i -X GET http://localhost:8001/routes
+```
+
+This request will return a `200` and a JSON response body object array with all of the routes configured on this {{site.base_gateway}} instance. 
+
+## Proxy a request 
+
+Kong is an API Gateway, it takes API calls from clients and routes them to the appropriate upstream application based on a series of configurations. Setting up services and routes models, on a small scale, how {{site.base_gateway}} works. Using the service and route that was configured you can now access `https://mockbin.org/` using `http://localhost:8000/mock`.
```

Comment:
```suggestion
Kong is an API Gateway, it takes API calls from clients and routes forwards them to the appropriate upstream application based on a series of configurations. Using the service and route that was previously configured, you can now access `https://mockbin.org/` using `http://localhost:8000/mock`.
```
I removed some text because it was repeating info that was previously mentioned. I think you could cut all but the last sentence if you wanted to (since the first sentence was already talked about in the overview of this doc).